test/attr/attr_nonEscapable.swift

```diff
@@ -0,0 +1,15 @@
+// RUN: %target-typecheck-verify-swift -enable-experimental-feature NonEscapableTypes
```

Comment:
This test seems to need a `REQUIRES:` to avoid the error:

```
error: experimental feature 'NonEscapableTypes' cannot be enabled in a production compiler
```

When trying to build and test a non-asserts compiler.

Reply:
Thanks. I really meant to do that.
https://github.com/apple/swift/pull/69268